src/nprintml/learn/automl.py

```diff
@@ -118,7 +120,7 @@ def graph_all(self, predictor, test_data):
         binarized_labels = binarizer.transform(y_true)
 
         y_pred = predictor.predict(test_no_label)
-        y_proba = predictor.predict_proba(test_no_label)
+        y_proba = predictor.predict_proba(test_no_label, as_multiclass=False)
```

Comment:
I wasn't 100% sure of this. It appears that `predict_proba` has been changed to give a `DataFrame` by default in this instance; and, perhaps that's "useful." Regardless, at least for now, this _appears_ to be the correct way to instruct the method to give us what we wanted before.

Reply:
i know we're testing this on the binary case in our test data. I don't have the link to the `as_multiclass=False` but does this break for a non-binary classification problem?

Reply:
It appears they made this change in order to make the output for binary look like the output for multiclass. So if anything it appears that we could omit this argument and then simplify _our_ code. (But I'm guessing we'd prefer to leave it as it is at least for now.)

```
as_multiclassbool, default = True

    Whether to return binary classification probabilities as if they were for multiclass classification.

        Output will contain two columns, and if as_pandas=True, the column names will correspond to the binary class labels. The columns will be the same order as predictor.class_labels.

    If False, output will contain only 1 column for the positive class (get positive_class name via predictor.positive_class). Only impacts output for binary classification problems.
```

```
Returns

    Array of predicted class-probabilities, corresponding to each row in the given data.
    May be a np.ndarray or pd.DataFrame / pd.Series depending on as_pandas and as_multiclass arguments and the type of prediction problem.
    For binary classification problems, the output contains for each datapoint the predicted probabilities of the negative and positive classes, unless you specify as_multiclass=False.
```